lib/ansible/galaxy/token.py

```diff
@@ -130,6 +130,10 @@ def _read(self):
 
         display.vvv('%s %s' % (action, to_text(self.b_file)))
 
+        if not isinstance(config, dict):
+            display.vvv('Failed to read %s as dictionary' % to_text(self.b_file))
```

Comment:
From the user point of view, I would suggest to not mention dictionary and say the file is not-properly formed 
"Galaxy token file %s malformed, unable to read it"

Reply:
Ok.